queries/sparql/sparql-18-b.sparql

```diff
@@ -0,0 +1,14 @@
+prefix example: <http://example/id/>
+PREFIX rdfs:<http://www.w3.org/2000/01/rdf-schema#>
+PREFIX rdf:<http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+select ?s ?p ?o {
+    # ok
+    ?s  rdf:type [ example:prop* ?o ] .
+    # ok
+    ?s  rdf:type [ example:prop* ?o ] ;
+        example:p2 ?_ .
+    # error 
```

Comment:
Should these comments be removed?

Reply:
Sorry, I forgot to clean that up. It's fixed now.